comps/agent/src/integrations/utils.py

```diff
@@ -66,7 +66,17 @@ def setup_chat_model(args):
             **params,
         )
     elif args.llm_engine == "openai":
-        llm = ChatOpenAI(model_name=args.model, request_timeout=args.timeout, **params)
+        if args.use_remote_service:
+            openai_endpoint = f"{args.llm_endpoint_url}/v1"
+            llm = ChatOpenAI(
+                openai_api_key=args.api_key,
```

Comment:
could we make the openai_api_key empty by default, so you don't need those if else in line 69 and 78. still passing openai_api_key and base but give empty string for none remote case.

Reply:
that could be easy solved with one line 
`openai_endpoint=args.llm_endpoint_url if args.llm_endpoint_url is None else args.llm_endpoint_url+ '/v1'`
also line 59-79 should be merged into one ChatOpenAI call